src/io.cpp

```diff
@@ -143,10 +142,10 @@ class File : public dap::ReaderWriter {
 
  private:
   FILE* const f;
+  const bool closable;
   std::mutex readMutex;
   std::mutex writeMutex;
-  std::atomic<bool> closed;
-  const bool closable;
+  std::atomic<bool> closed = { false };
```

Comment:
I think should be either {} or false, not both mixed together.

Reply:
`= false` does not compile. `{}` is not so explicit.

Reply:
Ah interesting, didn't realize std::atomic's copy ctor is deleted, and = false invokes that. Okay!